Song_sort_Genre.css

```diff
@@ -67,62 +41,152 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
-.content_1{
-	background-color: white;
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 5;
+}
+
+ul li ul li{
+	background-color:  #0C59CF;
+	display: block;
+}
+
+ul li ul li:hover{background: black;}
+
+ul li:hover ul{
+	display: block;
+	visibility: visible;
+}
+
+.logo img{
+	position:absolute;
+	margin-top:-20px;
+}
+
+
+
+.img_1{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
-	height: 100px;
-	border: 2px solid red;
 }
 
+.img_1 img{
+	display:block;
+	margin-left:auto;
+	margin-right:auto;
+	height:400px;
+	width:100%;
+}
+
+.footer{
+	background-color: #0C59CF;
+	grid-column-end: span 4;
+	padding-top:25px;
+	text-align: center;
+	color:white;
+	height:50px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+
+.content_1{
+	grid-column-start: 2;
+	grid-column-end: span 4;
+	background-color: white;
+	z-index: 2;
+	height:200px;
+	text-align:center;
+}
+
+.section3{
+	background-color: white;
+	width: 85%;
+	height: 200px;
+	position: absolute;
+	z-index: 4;
+}
+
+
+
+
+.section3 h1{
+	font-size:24px;
+	color:black;
+	margin-top:50px;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
+}
 
 .content_2{
 	grid-column-start: 1;
-	grid-column-end: span 1;
+	grid-column-end: 2;
 	background-color: white;
 	z-index: 2;
-	height:604px;
-	border: 2px solid red;
+	height:200px;
+}
+.content_2 img{
+	height:200px;
+	width:202px;
+	
 }
-
 
 .content_3{
-	grid-column-start: 2;
+	grid-column-start: 1;
```

Comment:
21/9: Unlike layout one layout two doesn't have a sidebar to I decided to make the table start at 1. I also set the background color to back and text to white because those two color was the color that I decided to me my main color and they display a great contrast.